test/test-sourcemapper.ts

```diff
@@ -234,11 +267,19 @@ testTool(
   ]
 );
 
+testTool(
```

Comment:
Is this the test that repros the original issue?  If so, please add some additional description or pointer to the bug because I'm a bit too dense to figure out what this is verifying.

Reply:
Added the comments.